README.md

```diff
@@ -74,6 +74,18 @@ Current guess at first steps (very open to debate):
 3. Build next-gen fontmake, converting parts to Rust and enabling parallelism
    * Guessing we would either do a good interchange format ([flatfont](https://github.com/googlefonts/flatfont)?) + ninja OR a Rust based executor using something like Rayon
 
+## Definitions
+
+### zerocopy
+
+As a shorthand, we use the term `zerocopy` to describe some desired properties of the font parsing API:
+
+* Reads of objects `n` bytes in size should consume no more than `n` bytes of font data
+* When possible, aggregates should be represented by a reference to a struct or slice that aliases font data
```

Comment:
Consider noting e.g. Rust equivalent to reinterpret_cast. I know it's redundant but I believe there are some readers (self) that would find this helpful

Reply:
In Rust this is called `transmute`. We can refer to the general idea as a bitcast. I'll add a note here.

Reply:
If I saw bitcast I think I'd have to look it up. If you said bitcast (e.g. transmute in Rust or reinterpret_cast in C++) you saved me a touch of time :)

Reply:
It’s LLVM (and SPIR-V) terminology so I view it as more general, but we’re targeting general programmers and not compiler engineers so point taken :)